Library/Formula/swift.rb

```diff
@@ -77,20 +79,19 @@ def install
         "--build-args=-j#{ENV.make_jobs}",
         "--lldb-use-system-debugserver",
         "--install-prefix=#{prefix}",
-        "--darwin-deployment-version-osx=#{MacOS.version}",
         "--build-jobs=#{ENV.make_jobs}"
     end
-    bin.install "#{build}/swift-macosx-x86_64/bin/swift",
-                "#{build}/swift-macosx-x86_64/bin/swift-autolink-extract",
-                "#{build}/swift-macosx-x86_64/bin/swift-compress",
-                "#{build}/swift-macosx-x86_64/bin/swift-demangle",
-                "#{build}/swift-macosx-x86_64/bin/swift-ide-test",
-                "#{build}/swift-macosx-x86_64/bin/swift-llvm-opt",
-                "#{build}/swift-macosx-x86_64/bin/swiftc",
-                "#{build}/swift-macosx-x86_64/bin/sil-extract",
-                "#{build}/swift-macosx-x86_64/bin/sil-opt"
-    (lib/"swift").install "#{build}/swift-macosx-x86_64/lib/swift/macosx/",
-                          "#{build}/swift-macosx-x86_64/lib/swift/shims/"
+    bin.install "#{build}/swift-linux-x86_64/bin/swift",
+                "#{build}/swift-linux-x86_64/bin/swift-autolink-extract",
+                "#{build}/swift-linux-x86_64/bin/swift-compress",
+                "#{build}/swift-linux-x86_64/bin/swift-demangle",
+                "#{build}/swift-linux-x86_64/bin/swift-ide-test",
+                "#{build}/swift-linux-x86_64/bin/swift-llvm-opt",
+                "#{build}/swift-linux-x86_64/bin/swiftc",
+                "#{build}/swift-linux-x86_64/bin/sil-extract",
+                "#{build}/swift-linux-x86_64/bin/sil-opt"
+    (lib/"swift").install "#{build}/swift-linux-x86_64/lib/swift/linux/",
+                          "#{build}/swift-linux-x86_64/lib/swift/shims/"
```

Comment:
To maintain compatibility with Mac,
```ruby
os = OS.mac? : "macosx" : OS::NAME
```
OS::NAME has either the value "darwin" on Mac or "linux" on Linux.
Then change
to
```ruby
"#{build}/swift-#{os}-x86_64/bin/swift",
```